src/components/ResultsCount/ResultsCount.tsx

```diff
@@ -26,7 +38,29 @@ export const ResultsCount = ({
           {currentCount} results of {resultsCount}
         </strong>
       </p>
-      <p className="cc-results-count__sort">Sorted by {sortedBy}</p>
+      {options ? (
+        <div className="cc-results-count__sort">
+          <Label
+            className="cc-results-count__label"
+            htmlFor={id}
+            text="Sort by:"
+          />
+          <select
+            className="cc-select cc-results-count__select"
+            id={id}
+            onChange={onChange}
+            value={selectedValue}
+          >
+            {options?.map(option => (
+              <option key={`option-${option.label}`} value={option.value}>
+                {option.label}
+              </option>
+            ))}
+          </select>
+        </div>
+      ) : (
```

Comment:
It should keep the logic for checking presence of sortedBy 

```js
  // ...
  ) : sortedBy && (
  // ...
```

Reply:
Minor: I don't think the outer parens (brackets) are needed

Reply:
I have just checked and it is added by prettier.